bluemira/codes/_freecadapi.py

```diff
@@ -232,13 +232,53 @@ def make_bezier(points: Union[list, np.ndarray], closed: bool = False) -> Part.W
 
 
 def make_bspline(
+    poles, mults, knots, periodic, degree, weights, check_rational
+) -> Part.Wire:
+    """
+    Builds a B-Spline by a lists of Poles, Mults, Knots
+
+    Parameters
+    ----------
+    poles: Union[list, np.ndarray]
+        list of poles.
+    multis: Union[list, np.ndarray]
+        list of integers for the multiplicity
+    knots: Union[list, np.ndarray]
+        list of knots
+    periodic: Bool
```

Comment:
`bool` should be lowercase here
```suggestion
    periodic: bool
```

Reply:
Again, as this function (and the version in the `geometry` module) is public, it would be good to expand on what this parameter does. Is it equivalent to the spline being closed?